src/pages/Settings/SavedSearchBeta.vue

```diff
@@ -38,51 +48,62 @@ export default {
 		SaveSearchItem,
 		WwwPage,
 		TheMyKivaSecondaryMenu,
+		KvButton,
 		KvGrid,
 		KvDefaultWrapper,
 	},
 	inject: ['apollo', 'cookieStore'],
 	data() {
 		return {
 			totalCount: 0,
-			savedSearches: () => []
+			savedSearches: [],
+			offset: 0,
+			limit: 40 // TODO: Limit is 40 because of an error via (totalCount) API
```

Comment:
What is the error?

Reply:
The totalCount from the API doesn't include ALL saved searches, it matches the limit set :(

Reply:
The error is that when the limit is reduced to a number under 10 (specifically 2) and the total count is over the limit, the total count displays as the limit instead of the actual amount